code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -247,6 +247,12 @@
 		if(hydration / max_hydration <= CREW_HYDRATION_VERYTHIRSTY)
 			msg += "[T.He] [T.is] severely dehydrated.\n"
 
+	if(moisture < 900)
+		msg += "<span class = 'warning'>[T.His] skin is flaking.</span>\n"

```

Comment:
This message may show to anyone examining any human mob, which seems pointless if it is a skrell related feature.